NEWS

```diff
@@ -1,3 +1,10 @@
+05-Jun-2017 IGNITION v0.17.1
+
+  Bug Fixes
+
+    - Fix issue preventing booting when user data was not provided on EC2 and
```

Comment:
`preventing booting` is an OS consequence, not an Ignition problem per se.  `Fix failure when...`?